models/shipping-method/src/shipping-rate/shipping-rate-draft/transformers.ts

```diff
@@ -0,0 +1,20 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TShippingRateDraft, TShippingRateDraftGraphql } from '../types';
+
+const transformers = {
+  default: Transformer<TShippingRateDraft, TShippingRateDraft>('default', {
+    buildFields: ['price', 'freeAbove', 'tiers'],
+  }),
+  rest: Transformer<TShippingRateDraft, TShippingRateDraft>('rest', {
+    buildFields: ['price', 'freeAbove', 'tiers'],
+  }),
+  graphql: Transformer<TShippingRateDraft, TShippingRateDraftGraphql>(
+    'graphql',
+    {
+      buildFields: ['price', 'freeAbove', 'tiers'],
+      addFields: () => ({ __typename: 'ShippingRateDraft' }),
```

Comment:
`typename` should be `ShippingRateInput`, I believe

Reply:
`ShippingRateDraft` is correct here.

![image](https://github.com/commercetools/test-data/assets/8136478/6d39395e-287a-4248-85e7-bb6391e9a58d)

Reply:
Ah yep, you're right! Sorry

Reply:
FWIW, I had the same thought when reviewing - I had to double check!